sound/soc/sof/control.c

```diff
@@ -489,3 +512,25 @@ int snd_sof_bytes_ext_get(struct snd_kcontrol *kcontrol,
 
 	return 0;
 }
+
+int snd_sof_control_type(type)
```

Comment:
how does this compile? you dont have a datatype for type?

Reply:
besides I think the argument should be called info and the return value should be called type?

Reply:
whoops...it is compiling as you see from CI results also... but yes don't know how and clearly a mistake, will fix.

Reply:
wow, maybe it compiles because the prototype has a type? funny...